go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/prebid/prebid-cache
 go 1.13
 
 require (
-	github.com/aerospike/aerospike-client-go v1.34.0
+	github.com/aerospike/aerospike-client-go v4.0.0+incompatible
```

Comment:
Update

Reply:
What does "incompatible" mean?

Reply:
Summarizing from [here](https://github.com/golang/go/wiki/Modules#can-a-module-consume-a-package-that-has-not-opted-in-to-modules), if our repository uses modules, every time we `go build` or `go test`:
 - new dependencies are automatically added to satisfy imports 
 - go.mod gets automatically updated 
 - new dependencies get automatically downloaded 

But if the package we just added has not opted into modules yet, but it is a v2+ package that has a valid semver tag, it gets automatically added into our go.mod with a `+incompatible` suffix